collectors/0/samza.py

```diff
@@ -101,5 +132,20 @@ def print_metric(metric_type, metric_name, ts, value, **tags):
         print ("%s.%s %d %s %s" % (metric_type.replace('org.apache.', ''), metric_name, ts, value, tags))
 
 
+def print_consumer_lag(ts, value, **tags):
+
+    if (unicode(str(value), 'utf-8').isnumeric()):
```

Comment:
This part also seems really similar to the function right above, I'd suggest consolidating the two.